tools/generate_schema_wrapper.py

```diff
@@ -888,6 +888,9 @@ def _create_encode_signature(
             # We could be more specific about what types are accepted in the list
             # but then the signatures would get rather long and less useful
             # to a user when it shows up in their IDE.
+
+            # NOTE: Currently triggered only for `detail`, `order`, `tooltip`
+            # I think the `tooltip` one especially we should be more specific
```

Comment:
**NOTE**: Currently triggered only for `detail`, `order`, `tooltip`
I think the `tooltip` one especially we should be more specific.